environment/src/main/resources/schema/app/20201208195427_CB-10242_API_changes_for_semi-private_networks.sql

```diff
@@ -0,0 +1,11 @@
+-- // CB-10242 API changes for semi-private networks
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE environment_network ADD COLUMN IF NOT EXISTS usepublicendpointaccessgateway BOOLEAN NOT NULL DEFAULT false;
```

Comment:
this fields can not be not null because in case of a production roll back we need to use the modified scheme with the previous CB version

Reply:
Default value shouldn't be added like this. It will block the whole table.
See https://dba.stackexchange.com/questions/60965/does-postgresql-optimize-adding-columns-with-non-null-defaults/216153#216153
Example: https://github.com/hortonworks/cloudbreak/blob/master/core/src/main/resources/schema/app/20201130155647_CB-9580_new_field_on_cluster:_embedded_db_is_configured_on_attached_disk_or_not.sql